nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -45,7 +45,7 @@ NOTE: When the endpoint configuration is not specified explicitly, the host scan
 
 ## Custom initialization and startup
 
-As of NServiceBus Version 5, customize the endpoint behavior using the `IConfigureThisEndpoint.Customize` method on the endpoint configuration class. Call the appropriate methods on the `BusConfiguration` parameter passed to the method.
+As of NServiceBus Version 5, customize the endpoint behavior using the `IConfigureThisEndpoint.Customize` method on the endpoint configuration class. Call the appropriate methods on the parameter passed to the method.
```

Comment:
instead of `As of NServiceBus Version 5` use the preferred version language. See https://github.com/particular/docs.particular.net#version-language

So, it would be: `For Versions 5 and above`